apps/tsdashboard/wristbandprinter.py

```diff
@@ -65,9 +91,34 @@ def setShowFlags(self, id, first, middle, father, mother, dob, gender,
         self._genderShow = gender
         self._curpShow = curp
 
+#
+# This procedure actually sends the ASCII data to the printer using lpr.
+# First open a temporary file in /tmp
+# Then send a newline and six spaces to the file.
+# Without the spaces, the patient ID is cut off.
+# Then send the wristband string to the file
+# Close the file.
+# The original print to the screen is still present.
+# Then, using the command lpr, print the file to the zebra printer
+#
+# Asumption: The same filename is used each time. The asumption is that lpr will
+# read in the file before another one is created. The printing is very fast, so
+# hopefully this will work. Otherwise, a temporary filename will be needed, and
+# cleanup will have to be done. This way no cleanup is needed.
+#
     def sendToPrinter(self):
+        import sys
+        import os
+
         wristbandStr = self.getUserFormattedWristband()
+        
+        f= open("/tmp/test.txt","w+")
+        f.write("\n      ")
+        f.write(wristbandStr)
+        f.close()
         print("{}".format(wristbandStr))
+        os.system("lpr -o orientation-requested=4 /tmp/test.txt")
```

Comment:
Suggest you factor this code, as it is platform specific, and would not work when run from a windows machine.

Reply:
I probably don't have the skillset to do this in python. In C, I would use #ifdef's